src/gensql/query/db.cljc

```diff
@@ -3,15 +3,14 @@
   of GenSQL databases."
   (:refer-clojure :exclude [empty read-string slurp])
   (:require #?(:clj [clojure.core :as clojure])
-            [borkdude.dynaload :as dynaload]
             [clojure.edn :as edn]
             [cognitect.anomalies :as-alias anomalies]
             [gensql.inference.gpm :as gpm]))
 
 (defn read-string
   [s]
-  (let [sppl-readers {'gensql.gpm.spe/SPE (dynaload/dynaload 'gensql.gpm.sppl/read-string)
-                      'inferenceql.gpm.spe/SPE (dynaload/dynaload 'gensql.gpm.sppl/read-string)} ; for backwards-compatibility
+  (let [sppl-readers {'gensql.gpm.spe/SPE (requiring-resolve 'gensql.gpm.sppl/read-string)
+                      'inferenceql.gpm.spe/SPE (requiring-resolve 'gensql.gpm.sppl/read-string)} ; for backwards-compatibility
```

Comment:
As mentioned in Slack, this won't work in cljs

Reply:
Okay @KingMob , i have resumed use of `dynaload`. Somehow the other patches to the build toolchain  preserved the right behavior, you're right, `requiring-resolve` must have been a green-herring .